src/httpSpansAgent.test.js

```diff
@@ -85,4 +87,20 @@ describe('HttpSpansAgent', () => {
 
     //no Error throwed
   });
+
+  test('postSpans - reject connection timout', async () => {
+    const reqBody = 'abcdefg';
+    globals.TracerGlobals.setTracerInputs({ token: 't_xxx' });
+
+    jest.spyOn(axios, 'create').mockImplementationOnce(() => {
+      return {
+        post: async (url, body, options) => {
+          expect(options.cancelToken).toBeDefined();
+          await sleep(500);
+        },
+      };
+    });
+
+    await HttpSpansAgent.postSpans(reqBody);
```

Comment:
Make sure that `axios. create. post` was called.
You can do it with a variable being updated in the function or with jest.